src/lib.rs

```diff
@@ -1182,6 +1182,7 @@ impl<'a> ParserState<'a> {
                 b'J' => Type::Int64(sc),
                 b'K' => Type::Uint64(sc),
                 b'W' => Type::Wchar(sc),
+                b'Q' => Type::Char(sc),
```

Comment:
This is missing, I added it into the same PR and forgot about it. We can merge it together or I back this out.